locale/fi/LC_MESSAGES/django.po

```diff
@@ -3443,6 +3451,58 @@ msgstr ""
 "Ilmoitusviestien vastaanottajat. Esimerkki: matti@example.com,maija@example."
 "com"
 
+msgctxt "Search class"
+msgid "Plot search"
+msgstr "Tonttihaku tai kilpailu"
```

Comment:
I've been under the impression that the intention here is more towards "Tonttihaku tai tonttikilpailu", with the latter shortened for brevity, so "Tonttihaku tai -kilpailu". That form has been used in the public UI as well (though I did find an instance of this dashless form in use as well in a component from before our time, which should probably be eventually fixed).